catena4450m101_sensor/application_commands.cpp

```diff
@@ -0,0 +1,95 @@
+//
```

Comment:
This really has nothing to do with water level. The changes for the added command need to be in a different pull request.

Reply:
Sure, I will pull the master and add only water level sensing related changes. Will do adding commands changes as separate pull request.